js_modules/dagit/packages/core/src/app/Util.tsx

```diff
@@ -6,6 +6,29 @@ function twoDigit(v: number) {
   return `${v < 10 ? '0' : ''}${v}`;
 }
 
+function indexesOf(string: string, regex: RegExp) {
+  const indexes: number[] = [];
+  let match = null;
+  while ((match = regex.exec(string))) {
```

Comment:
someone on StackOverflow pointed out (i had no idea wut this syntax was even doing) that in the absence of a /g in the regex this will run an infinte loop and never exit...might want to put a check in here or add the g flag yourself?

Reply:
Ooof that's super interesting - yeah I think this function should apply the `g` flag to the regexp before doing this 😬